packages/playground/client/src/index.ts

```diff
@@ -94,6 +95,7 @@ export async function startPlaygroundWeb({
 		}),
 		progressTracker
 	);
+	get_logger().addPlaygroundRequestEndListener(playground);
```

Comment:
I'm fine with merging this as is and iterating in another PR, but just to start this discussion – what do you think about the following API?

```suggestion
// This could be moved to some util package OR the logger
// package could import it directly and just know which runtime
// is it being ran at as it might need more nuanced detection
// than 'WEB' | 'NODE' | 'WORKER'
import { currentJsRuntime } from '@php-wasm/universal'; 

// A universal error logger for any PHP.wasm instance
import { ErrorLogger } from '@php-wasm/logger';

ErrorLogger.attach({
	// Playground is just a UniversalPHP instance
	php: playground,
	
	// Avoid hardcoding the WordPress-specific path in a
	// generic logger
	errorLogPath: '/wordpress/wp-content/debug.log',
	
	// Tell the logger which set of JS error events to bind to
	jsRuntime: currentJsRuntime
});
```

Reply:
> import { ErrorLogger } from '@php-wasm/logger';

Should the logger live in php-wasm or Playground?

From our discussion yesterday, I think that Playground is the better place.
This allows us to still access logs if PHP-wasm crashes.


> what do you think about the following API?

Seems, good, I'm just curious, when would we attach multiple "clients"?

Reply:
> jsRuntime: currentJsRuntime

How would this option work?

Reply:
> From our discussion yesterday, I think that Playground is the better place.
> This allows us to still access logs if PHP-wasm crashes.

Oh, I'm not suggesting funneling the logs into the PHP.wasm instance. I stand by all the thoughts we've exchanged about the data flow. In here, I only mean choosing the `@php-wasm` package prefix rather than `@wp-playground` to make the logic reusable in all PHP.wasm apps, like `wp-now` vs making it only available for the in-browser Playground.

Reply:
I moved it now and added all the package boilerplate. [b14f4a5](https://github.com/WordPress/wordpress-playground/pull/1035/commits/b14f4a520ef31d306170254a5f5a2786abaa128a)

This also allowed me to add it to Node.